code-challenges.js

```diff
@@ -6,55 +6,159 @@
+//     else if (temp > boilingPoint) {
+//         return `${temp} is above boiling point`;
+//         }
+//         else {
+//             return `${temp} is at boiling point`
+//         }
+//     }
+
+//tried ternary operators for experience and shorter code/cleaner? 
+const testTemp = (temp) => {
+    let boilingPoint = 212;
+    let result = "";
+
+return result = temp < boilingPoint ? `${temp} is below boiling point` : (temp > boilingPoint ? `${temp} is above boiling point` : `${temp} is at boiling point`)
+    }
+
+console.log(testTemp(temp1));
+console.log(testTemp(temp2));
+console.log(testTemp(temp3));
+
+//---------------------------------------------------------------------------------------------------------
 // --------------------2) Create a function that multiplies each number in the array by 5 using a for loop.
 // Use the test variable provided below. Expected outcome: [15, 35, 0, 30, -45]
 
 var myNumbers1 = [3, 7, 0, 6, -9]
+//create a function called multiplyByFive
+//make var called array which stores an empty array
+//iterate through array and multiply by 5
+//push to new array
 
+const multiplyByFive = (num) => {
+ let array = [];
+    for (let i = 0; i < num.length; i++) {
+        array.push(num[i] * 5)
+    }
+    return array;
+}
 
-
+console.log(multiplyByFive(myNumbers1));
 
 // --------------------3) Create a function that multiplies each number in the array by 5 using map.
 // Use the test variable provided below. Expected outcome: [40, -35, 0, 30, 10]
 
 var myNumbers2 = [8, -7, 0, 6, 2]
+//create a function called mapFive 
+//use map to iterate through myNumbers2 and multiply each value by 5
 
 
+const mapFive = myNumbers2.map(val => val * 5); //had to refer to syllabus for syntax set up..
+//console.log(mapFive(myNumbers2)) //this did not work because we saved it into a variable called mapFive so when I tried to keep calling it I got is not a function.. 
+console.log(mapFive)
 
-
+//---------------------------------------------------------------------------------------------------------
 // --------------------4) Create a function that removes all the vowels from a string.
 // Use the test variables provided below. Expected output: "HyThrLrnStdnt" "LvJvScrpt"
 
 var stringWithVowels1 = "HeyThereLearnStudent"
 var stringWithVowels2 = "ILoveJavaScript"
+//create a function called disemvowel that takes in one parameter str
+//it should remove all vowels from a string called
+//create an array called vowels that contains the vowels
+//split the str so we can check each letter individually
+//use filter to check each (char)  
+//join the results
 
+const disemvowel = (str) => {
+    let vowels = ['a', 'e', 'i', 'o', 'u'];
+    let splitString = str.split("");
+    //console.log(splitString);
+    return splitString
+    .filter(char => !vowels.includes(char.toLowerCase()))//when I put .join("") here without breaking it into another line it was giving me an error because i did !vowels.includes(char).join(""))
```

Comment:
starting a new line in js doesn't actually break up this code maybe how you're thinking. you just need to append the `.join("")` to the closing `)` of the filter. this code runs:

`vowels.includes(char.toLowerCase())).join("")` :) 

great solution!